src/components/desktop/SidePane.js

```diff
@@ -1,15 +1,12 @@
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
 
-import Search from '../search/Search'
+import SearchOverlay from './SearchOverlay'
 
-const PaneContainer = styled.div`
-  padding: 20px 10px;
-  padding-right: 0px;
-`
+const PaneContainer = styled.div``
```

Comment:
Since `PaneContainer` is has an empty ruleset could we make a div instead of defining a new styled-component?

Reply:
Good point - changed to a fragment. Left that there bc I thought it might need styles later but yeah seems better to take it out for now